ui/app/AppLayouts/Communities/popups/CreateChannelPopup.qml

```diff
@@ -70,293 +178,456 @@ StatusDialog {
-                              colorDialog.color.toString().toUpperCase() :
-                              qsTr("Pick a colour")
+                Rectangle {
+                    Layout.fillWidth: true
+                    Layout.fillHeight: true
+                    color: Theme.palette.baseColor4
+
+                    StatusListView {
+                        anchors.fill: parent
+                        anchors.margins: 16
+                        model: root.communitiesStore.discordCategoriesModel
+                        delegate: ColumnLayout {
+                            width: ListView.view.width
+                            spacing: 8
+
+                            StatusBaseText {
+                                readonly property string categoryId: model.id
+                                id: categoryCheckbox
+                                text: model.name
+                            }
+
+                            ColumnLayout {
+                                spacing: 8
+                                Layout.fillWidth: true
+                                Layout.leftMargin: 24
+                                Repeater {
+                                    Layout.fillWidth: true
+                                    model: root.communitiesStore.discordChannelsModel
+                                    delegate: StatusRadioButton {
+                                        width: parent.width
+                                        text: model.name
+                                        checked: model.selected
+                                        visible: model.categoryId === categoryCheckbox.categoryId
+                                        onToggled: root.communitiesStore.toggleOneDiscordChannel(model.id)
+                                    }
+                                }
+                            }
+                        }
+                    }
+                }
+            }
+        }
+    ]
 
-                    onClicked: colorDialog.open();
-                    onTextChanged: {
-                        if (colorDialog.colorSelected) {
-                            validationError = Utils.validateAndReturnError(text, communityColorValidator)
+    Connections {
+        enabled: root.opened && root.emojiPopupOpened
+        target: emojiPopup
+
+        function onEmojiSelected(emojiText: string, atCursor: bool) {
+            nameInput.input.asset.isLetterIdenticon = false;
+            nameInput.input.asset.emoji = emojiText
+        }
+        function onClosed() {
+            root.emojiPopupOpened = false
+        }
+    }
+
+    stackItems: [
+        StatusScrollView {
+            id: scrollView
+
+            readonly property bool canGoNext: d.isFormValid()
+
+            property ScrollBar vScrollBar: ScrollBar.vertical
+
+            contentWidth: availableWidth
+            padding: 0
+
+            function scrollBackUp() {
+                vScrollBar.setPosition(0)
```

Comment:
shouldn't it be just
```suggestion
                vScrollBar.position = 0
```
?
I can't find setPosition in a documented api

Reply:
Not my code, dunno

Reply:
yeah, but maybe good time to fix it once spotted

Reply:
Probably some copy&pasted code (not mine :), but it definitely works